gensrc/thrift/FrontendService.thrift

```diff
@@ -324,6 +324,14 @@ struct TTableStatus {
     5: optional i64 last_check_time
     6: optional i64 create_time
     20: optional string ddl_sql
+    // id for materialized view
+    21: optional string id
+    // database_name for materialized view
+    22: optional string database_name
+    // ddl_sql for materialized view
+    23: optional string text
```

Comment:
for the  information_schema.views, we use ddl_sql, I think for  materialized view, we also could use ddl_sql.